Accessing images for analysis.ipynb

```diff
@@ -0,0 +1,5097 @@
+     "text": [
+      "[\n",
+      "  \"<p>The Library of Congress does not&nbsp;own rights to material  in its collections. Therefore, it does not license or charge permission fees  for use of such material and cannot grant or deny permission to publish or  otherwise distribute the material. </p>\\n<p>Ultimately, it is the researcher's obligation to assess copyright or other use restrictions and  obtain permission from third parties when necessary before publishing or  otherwise distributing materials found in the Library's collections. </p>\\n<p>For information about reproducing,  publishing, and citing material from this collection, as well as access to the  original items, see: <a href=\\\"//www.loc.gov/rr/print/res/217_wpa.html\\\">Work Projects Administration Posters - Rights and Restrictions Information</a> </p>\\n \\n\"\n",
+      "]\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(json.dumps(r_data[\"item\"][\"rights\"], indent=2))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**restriction**\n",
+    "\n",
+    "Restriction is often blank."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\"\"\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(json.dumps(r_data[\"item\"][\"restriction\"], indent=2))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Accessing images\n",
+    "\n",
+    "Now that you have the URLs for images, you can access them as part of your image analysis or download them. Just be sure to check out the rights and restrictions information about how you can distribute or display them.\n",
+    "\n",
+    "The code below will acccess and save the images located the URLs you saved earlier. It saves them to a directory, so if you haven't already created a directory where you want to save the files, do that now. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "!mkdir images"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "def get_image_files(image_urls_list, path):\n",
+    "    '''\n",
+    "    Takes as input a list of URLs for loc.gov item pages and \n",
+    "    a path to a directory in which to save image files, e.g. \"data\". \n",
+    "    '''    \n",
+    "    for count, url in enumerate(image_urls_list):\n",
+    "        if count % 100 == 0:\n",
+    "            print(\"at item {0}\".format(count))\n",
+    "        try:\n",
+    "            #filename = create a filename based on taking part of the URL and substituting - for / in path.\n",
+    "            filename = url.split('/')[-1]\n",
```

Comment:
this will fail if there is a dangling `/` on the url. Might be safer to use `urlparse` https://docs.python.org/2/library/urlparse.html

Reply:
The URLs in the image_url path always end in a filename, but I think that this could be a problem with the other function, get_and_save_images, which is parsing the id field. Thanks for catching this. 

Will squash commits and update.